src/components/Tooltip.tsx

```diff
@@ -29,7 +29,7 @@ export default function Tooltip({
   return (
     <div
       // @ts-ignore while awaiting merge of this PR for support of "inert" https://github.com/DefinitelyTyped/DefinitelyTyped/pull/60822
-      inert={`${inert}`}
+      inert={inert}
```

Comment:
I merged this accidently earlier,

But either way it's weird, it leave as really gross error in the console
![image](https://github.com/user-attachments/assets/3df65fd9-0b67-4fe0-a066-150c54fa69c6)


But if you try and do what it asks and turn it into a string, it causes the tool tip to disappear too quickly, so I just changed it back.